src/latis-python-client/latis.py

```diff
@@ -1,10 +1,28 @@
+"""
+This script allows for retrieving and manipulating datasets
+from a Latis instance. It provides functionality to retrieve
+the dataset in either numpy or pandas format, and optionally
+download the dataset to a file.
+"""
```

Comment:
"The module-level docstring at the start of your script is well written, but according to PEP 257, it should be a one-liner followed by a more detailed description if necessary. Something like:
```
"""
A script to retrieve and manipulate datasets from a Latis instance.

This script provides functionality to retrieve the dataset in either numpy 
or pandas format, and optionally download the dataset to a file.
"""
```
"